package/src/utils/utils.ts

```diff
@@ -350,6 +354,41 @@ export type QueryMembersFunction<
   },
 ) => Promise<void>;
 
+/**
+ * Default emoji search index for auto complete text input
+ */
+export const defaultEmojiSearchIndex: EmojiSearchIndex = {
+  search: (query) => {
+    const result = compiledEmojis.reduce((acc, cur) => {
```

Comment:
```
compiledEmojis.reduce
```

I know that this is older code. But its inefficient in my view

due to `if (acc.length >= 10) return acc;`, we dont early exit after this condition but still keep looping.

If a simple for loop is used we can mitigate this.

Can you change it pls.

Reply:
Done